packages/flutter/lib/src/cupertino/interface_level.dart

```diff
@@ -0,0 +1,76 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Are we supposed to update these dates to 2019 when we create a new file?  I'm wondering out loud, I'm unsure.

Reply:
Also `The Chromium Authors` vs `The Flutter Authors`, which one should this file use?

Reply:
The flutter/flutter repo should be the The Chromium Authors. Use the current year unless you've copied the code from an existing file

Reply:
This is a new file, the year should be 2019